doc/userGuide.md

```diff
@@ -338,6 +338,18 @@ To exclude the test case with variation ```-Xmx1024m```:
 
 ```auto exclude test jdk_test_1```
 
+To exclude the test for openj9 only:
+
+```auto exclude test jdk_test impl=openj9```
+
+To exclude the test for java 8 only:
+
+```auto exclude test jdk_test ver=8```
+
+To exclude the test case with variation ```-Xmx1024m``` against openj9 java 8 only:
```

Comment:
It may be useful to say "To exclude the 2nd variation listed which is assigned suffix _1 ```-Xmx1024m```  against openj9 java 8 only:"

Reply:
Updated